jekyll/content/terms/generative-adversarial-network-gan.md

```diff
@@ -5,3 +5,9 @@ references:
  - "[Generative adversarial networks](https://arxiv.org/abs/1406.2661)"
  - "[NIPS 2016 tutorial: Generative Adversarial Networks](https://arxiv.org/abs/1701.00160)"
 ---
+Generative adversarial networks were proposed by Ian Goodfellow in 2014. A GAN consists of a model with two neural networks, G (generative model) and D (discriminative model), which are competing in a minimax two-player game.
```

Comment:
GANs are a tricky topic. I avoided filling in this definition because it is such a difficult topic. :)

But I think a really good definition for GANs will highlight on motivations:

 - Why do GANs matter? We use them for certain generative tasks like image completion and superscalar resolution. [More details here](https://www.cs.toronto.edu/~duvenaud/courses/csc2541/slides/gan-applications.pdf).

The core of GANs is that they are two neural networks... jointly trained... with a loss function that represents a minimax game between the two networks.

I think it's a difficult definition to write, and it might make sense to make the GAN definition into a separate pull request if you're interested. I'm happy to help you write it.

Reply:
Some improvements and suggestions I have for this article is to:

 - Include and explain the loss function from [the original GAN paper](https://arxiv.org/abs/1406.2661). This might be a little bit much... since we don't do that for other definitions, but when I first started studying GANs, I didn't understand them until I broke down the loss function.

 - Figure out how to describe more clearly that the generator network generates fakes and samples from the training distribution, and the discriminator's job is to tell them apart. You kinda do this... but I'm struggling with a way to rephrase it right now. I know, that's a lame criticism. :)

Reply:
I know that's a lot of feedback, but this is also why I haven't bothered filling out this definition either...